README.md

```diff
@@ -82,18 +82,25 @@ See [CONTRIBUTING wiki](https://github.com/oneapi-src/oneAPI-samples/blob/master
 |2021.4.0|[Pub: Data Parallel C++:](https://www.apress.com/9781484255735)<br>[Chapter 20 - Epilogue Future Direction](https://github.com/oneapi-src/oneAPI-samples/tree/master/Publications/DPC++/Epilogue_future_direction)|Collection of Code samples for the chapterEpilogue source code examples: Future Direction of DPC++  <br>- Fig_ep_1_mdspan.cpp - Attaching accessor-like indexing to a USM pointer using mdspan  <br>- Fig_ep_2-4_generic_space.cpp - Storing pointers to a specific address space in a class - Figure EP-3. Storing pointers to the generic address space in a class - Figure EP-4. Storing pointers with an optional address space in a class  <br>- Fig_ep_5_extension_mechanism.cpp - Checking for Intel sub-group extension compiler support with #ifdef  <br>- Fig_ep_6_device_constexpr.cpp - Specializing kernel code based on device aspects at kernel compile time  <br>- Fig_ep_7_hierarchical_reduction.cpp - Using hierarchical parallelism for a hierarchical reduction|
+|2021.4.0|[Intel(R) Extension for Scikit-learn: SVC for Adult dataset](https://github.com/oneapi-src/oneAPI-samples/tree/master/AI-and-Analytics/Features-and-Functionality/IntelScikitLearn_Extensions_SVC_Adult)|Use Intel(R) Extension for Scikit-learn to accelerate the training and prediction with SVC algorithm on Adult dataset. Compare the performance of SVC algorithm optimized through Intel(R) Extension for Scikit-learn against original Scikit-learn.|
 |2021.4.0|[Intel® Python Scikit-learn Extension Getting Started](https://github.com/oneapi-src/oneAPI-samples/tree/master/AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_SKLearn_GettingStarted)|This sample illustrates how to do Image classification using SVM classifier from Python API package SKlearnex with the use of Intel® oneAPI Data Analytics Library (oneDAL).|
 |2021.4.0|[Merge Sort](https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort)|A Reference design demonstrating merge sort on an Intel® FPGA|
 |2021.4.0|[Private Copies](https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/DPC++FPGA/Tutorials/Features/private_copies)|An Intel® FPGA tutorial demonstrating how to use the private_copies attribute to trade off the resource use and the throughput of a DPC++ FPGA program|
 |2021.4.0|[Stall Enable](https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/DPC++FPGA/Tutorials/Features/stall_enable)|An Intel® FPGA tutorial demonstrating the use_stall_enable_clusters attribute|
 
 Total Samples: 158
 
+
+## Deleted Code Samples
```

Comment:
This is showing as 2021..Gold, when in fact I believe it was 2021.1.Gold-2021.3.  We should also show the path if we are going to suggest a previous version  DirectProgramming/DPC++FPGA/Tutorials/Tools/use_library/CMakeLists.txt and the previous version should include the hyperlink to its GitHub tag aka https://github.com/oneapi-src/oneAPI-samples/releases/tag/2021.3.0

Please use 2022.3, should read as 2021.3